image/base/install-oxipng

```diff
@@ -18,7 +18,11 @@ echo "${OXIPNG_HASH} ${OXIPNG_ARCHIVE}" | sha256sum -c
 tar -zxf ${OXIPNG_ARCHIVE}
 cd ${OXIPNG_DIR}
 
-/usr/local/cargo/bin/cargo build --release
+if [ $(dpkg --print-architecture) == "amd64" ]; then
+  /usr/local/cargo/bin/cargo build --release
+else
+  /usr/local/cargo/bin/cargo build --release --jobs 1 --verbose
+fi
```

Comment:
I found a post suggesting setting `RUST_LOG=cargo=debug`, so maybe some output from

```
RUST_LOG=cargo=debug /usr/local/cargo/bin/cargo build --release --jobs 1 --verbose
```

Reply:
Thanks! I think I fixed the issue by using a feature from the brand-new Rust 1.68. I've updated the PR and we will know for sure in a couple of hours.